Dockerfile-interactive

```diff
@@ -0,0 +1,36 @@
+# Pull main environment
+ARG PYTHON_VERSION=3.8.10
+FROM python:$PYTHON_VERSION AS base
+
+ENV PYTHONUNBUFFERED=1
```

Comment:
PYTHON_VERSION variable was added to `.env` file. Since it is required by `FROM` procedure in Dockerfile, it needs to be passed as `args` in `docker-compose.yaml`.